packages/client/tests/e2e/driver-adapters-accelerate/tests/url.ts

```diff
@@ -0,0 +1,45 @@
+test('driver adapters cannot be used with accelerate via @prisma/client/wasm', () => {
+  jest.isolateModules(() => {
+    expect.assertions(1)
+    const { PrismaClient } = require('@prisma/client/wasm')
+
+    const newClient = () =>
+      new PrismaClient({
+        adapter: {
+          queryRaw: () => {},
+          executeRaw: () => {},
+          startTransaction: () => {},
+        },
+        datasourceUrl: 'prisma://localhost:1234',
+      })
+
+    expect(newClient).toThrowErrorMatchingInlineSnapshot(`
+"Prisma Client was configured to use the \`adapter\` option but the URL was a \`prisma://\` URL.
+Please either use the \`prisma://\` URL or remove the \`adapter\` from the Prisma Client constructor."
+`)
+  })
+})
+
+test('driver adapters cannot be used with accelerate via @prisma/client/default', () => {
+  jest.isolateModules(() => {
+    expect.assertions(1)
```

Comment:
I'm not sure why `expect.assertions(1)` is actually needed here?

Reply:
Left over from another test construct, thanks for catching